dev/com.ibm.ws.jpa.tests.eclipselink_32_fat.common/bnd.bnd

```diff
@@ -0,0 +1,62 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+test.project: true
+publish.wlp.jar.disabled: true
+instrument.disabled: true
+
+# To define a global minimum java level for the FAT, use the following property.
+# If unspecified, the default value is ${javac.source}
+#fat.minimum.java.level: 17
+
+javac.source: 17
+javac.target: 17
+
+src: \
+    fat/src,\
+    test-applications/common/src,\
+    test-applications/example/src,\
+    test-applications/olgh8014/src,\
+    test-applications/olgh8294/src,\
+    test-applications/olgh8461/src,\
+    test-applications/olgh8950/src,\
+    test-applications/olgh9018/src,\
+    test-applications/olgh9035/src,\
+    test-applications/olgh10068/src,\
+    test-applications/olgh14426/src,\
+    test-applications/olgh14457/src,\
+    test-applications/olgh16588/src,\
+    test-applications/olgh16685/src,\
+    test-applications/olgh16772/src,\
+    test-applications/olgh16970/src,\
+    test-applications/olgh19176/src,\
+    test-applications/helpers/DatabaseManagement/src
+
+# Dependencies may be local bundles (e.g. com.ibm.websphere.javaee.servlet.3.1)
+#      or binaries from Artifactory (e.g. commons-logging:commons-logging)
+-buildpath: \
```

Comment:
Similar to the other 31 and 32 common projects, the buildpath should contain the common... so should include:

com.ibm.ws.jpa.tests.eclipselink_fat.common

Adding this to buildpath will avoid the need to copy SQLListener.